tests/fixtures/mod.rs

```diff
@@ -42,23 +48,42 @@ pub fn tmpdir() -> TempDir {
     let tmpdir = assert_fs::TempDir::new().expect("Couldn't create a temp dir for tests");
     let mut files = FILES.to_vec();
     files.extend_from_slice(HIDDEN_FILES);
-    for file in &files {
+    let (files_hard, files_symlink) = files.split_at(1);
+    println!("path = {:?}", tmpdir.path());
+    for file in files_hard {
         tmpdir
             .child(file)
             .write_str("Test Hello Yes")
             .expect("Couldn't write to file");
     }
+    for file in files_symlink {
```

Comment:
Could you sprinkle a few comments in here about what we're doing in all the loops and such now that this fixture has become a bit more complex?